css/properties/border-collapse.json

```
@@ -70,9 +70,11 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "1.2"
```

I have already seen it in Safari 1.1 (see [here](https://github.com/caugner/css-properties/blame/c2748bdba6c0f6a47de7e54d7cdde53bd967622c/properties.css#L13-L13C16)):

```suggestion
                "version_added": "1.1"
```